src/PactNet.Native/PactExtensions.cs

```diff
@@ -49,6 +49,25 @@ public static IPactBuilderV3 UsingNativeBackend(this IPactV3 pact, int? port = n
             return builder;
         }
 
+        /// <summary>
+        /// Establish a new pact using the native backend
+        /// </summary>
+        /// <param name="pact">Pact details</param>
+        /// <returns>Pact builder</returns>
+        /// <remarks>
+        /// If multiple mock servers are started at the same time, you must make sure you don't supply the same port twice.
```

Comment:
Is this comment accurate? Message pacts don't create mock servers do they?

Reply:
copy paste ^^